Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -399,6 +559,171 @@ def get_scan_status_command():
+            results.append(get_entry_for_object("The requested scan was paused successfully",
+                                                "TenableIO.Scan", replace_keys(resumed_scan), ["Id", "Status"]))
+
+        else:
+            results.append(
+                "Command 'tenable-io-pause-scans' cannot be "
+                "called while scan status is {} for scanID {}".format(scan_status["Status"], scan_id))
+
+    return results
+
+
+# TEST
+def resume_scan_command():
+    """Function for integration command resume_scan_command."""
+    scan_id = demisto.getArg('scanId')
+    scan_details = send_scan_request(scan_id)
+    scan_status = {
+        'Id': scan_id,
+        'Status': scan_details['info']['status']
+    }
+
+    if scan_status["Status"].lower() == "paused":
+        send_scan_request(scan_id, "resume", "POST")
+        resumed_scan = {
+            "Id": scan_id,
+            "Status": "Resuming"
+        }
+        return get_entry_for_object(
+            "The requested scan was resumed successfully", "TenableIO.Scan", replace_keys(resumed_scan), ["Id", "Status"])
+
+    else:
+        return "Command 'tenable-io-resume-scan' cannot be called while scan status is {}".format(scan_status["Status"])
+
+
+def resume_scans_command():
+    """Function for integration command resume_scans_command."""
+    scan_ids = str(demisto.getArg('scanIds')).split(",")
+
+    results = []
+
+    for scan_id in scan_ids:
+
+        scan_details = send_scan_request(scan_id)
+        scan_status = {
+            'Id': scan_id,
+            'Status': scan_details['info']['status']
+        }
+
+        if scan_status["Status"].lower() == "paused":
+            send_scan_request(scan_id, "resume", "POST")
+            resumed_scan = {
+                "Id": scan_id,
+                "Status": "Resuming"
+            }
+            results.append(get_entry_for_object("The requested scan was resumed successfully",
+                                                "TenableIO.Scan", replace_keys(resumed_scan), ["Id", "Status"]))
+
+        else:
+            results.append(
+                "Command 'tenable-io-resume-scans' cannot be "
+                "called while scan status is {} for scanID {}".format(scan_status["Status"], scan_id))
+
+    return results
+
+def get_scan_templates():
+    """Function for integration command get_scan_templates."""
+    try:
+        endpoint = BASE_URL + "editor/scan/templates"
+        response = requests.request("GET", endpoint, headers=NEW_HEADERS, verify=USE_SSL)
+        response.raise_for_status()
+        #print(response.json())
+        demisto.info("Ran request sucessfully")
+        return response
+    except HTTPError:
+        #print(response.status_code)
+        demisto.error(traceback.format_exc())
+        sys.exit(0)
+
+
+# Request/Response methods
+# kwargs: request parameters
+def send_request(payload, endpoint="", method='GET', endpoint_base="tags", ignore_license_error=False, **kwargs):
```

Comment:
You should merge `send request` and `send_scan_request` into one function and add a flag that says if it is a `scan-request` or not.